lib/ansible/modules/database/influxdb/influxdb_user.py

```diff
@@ -0,0 +1,206 @@
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: influxdb_user
+short_description: Manage InfluxDB users
+description:
+    - Manage InfluxDB users
+version_added: 2.5
+author: "Vitaliy Zhhuta (@zhhuta)"
+requirements:
+    - "python >= 2.6"
+    - "influxdb >= 0.9"
+options:
+    hostname:
+        description:
+            - The hostname or IP address on which InfluxDB server is listening
+        required: false
+    username:
+        description:
+            - User that we want to create
+        default: None
+        required: True
+    password:
+        description:
+            - Password that we want to set for user
+        default: None
+        required: false
+    admin:
+        description:
+            - specify if user should be admin
+        default: False
+        required: false
+
+    port:
+        description:
+            - The port on which InfluxDB server is listening
+        default: 8086
+        required: false
+    state:
+        description:
+            - Determines if the database should be created or destroyed
+        choices: ['present', 'absent']
+        default: present
+        required: false
+    login_username:
+        description:
+            - user to auth wirh influxdb
+        default: root
+        required: false
+    login_password:
+        description:
+            - user to auth wirh influxdb
+        default: root
+        required: false
+
+'''
+
+EXAMPLES = '''
+# Example influxdb_database command from Ansible Playbooks
+- name: Create User
+  influxdb_user:
+      username: "{{influxdb_user_name}}"
+      password: "{{influxdb_user_password}}"
+      state: present
+
+- name: Destroy User
+  influxdb_database:
+      username: "{{influxdb_user_name}}"
+      password: "{{influxdb_user_password}}"
+      state: absent
+
+- name: Create user on dest host
+  influxdb_database:
+      hostname: "{{influxdb_ip_address}}"
+      login_username: "{{influxdb_username}}"
+      login_password: "{{influxdb_password}}"
+      username: "{{influxdb_user_name}}"
+      password: "{{influxdb_user_password}}"
+      admin: true
+      state: present
+'''
+
+RETURN = '''
+#only defaults
+'''
+
+try:
+    import ansible.module_utils.urls
+    from influxdb import InfluxDBClient
+    from influxdb import exceptions
+
+    HAS_INFLUXDB = True
+except ImportError:
+    HAS_INFLUXDB = False
+
+from ansible.module_utils.basic import AnsibleModule
+
+
+def influxdb_argument_spec():
+    return dict(
+        hostname=dict(default='localhost', type='str'),
+        port=dict(default=8086, type='int'),
+        login_username=dict(default='root', type='str'),
+        login_password=dict(default='root', type='str', no_log=True),
+        username=dict(required=True, type='str'),
+        password=dict(required=False, type='str', no_log=True),
+        admin=dict(default='False', type='str')
```

Comment:
make this a boolean 
~~~diff
-        admin=dict(default='False, type='str')
+        admin=dict(default=False, type='bool')
~~~